network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -126,7 +126,7 @@ class InitiativesPage(PrimaryPage):
     ]
 
 
-class ParticipatePage2(PrimaryPage):
+class ParticipatePage(PrimaryPage):
     parent_page_types = ['Homepage']
     template = 'wagtailpages/static/participate_page2.html'
```

Comment:
this has been left pointing to the original template until things actually work.